### ISSUE TITLE
Clarification: Does keras_tuner support inputting prior hyperparameter data?

### ISSUE THREAD

**Issue by nilamandal** [keras-team/keras-io#1215]
Hello, I am trying to clarify whether or not keras_tuner objects such as the BayesianOptimization tuner have the ability to take as input prior hyperparameter combinations (for example, from a gridwise search conducted before running the Keras hyperparameter tuner).

Does this functionality exist in the current body keras_tuner?

Thank you!

**Comment by rchao** [comment]
Haifeng, would you have an answer for this?

**Comment by sachinprasadhs** [comment]
@haifeng-jin

**Comment by maitry63** [comment]
Hi @nilamandal , Right now, as per there is no built-in functionality in the current keras_tuner that allows you to feed in prior hyperparameter combinations and outcomes to seed or warm-start BayesianOptimization or other tuners.

The BayesianOptimization tuner does not accept prior results as input. Instead, the only relevant parameters related to initialization are:

- num_initial_points: number of randomly generated points to use for the initial phase of Bayesian optimization.
- hyperparameters: a HyperParameters object that can pre-register the search space definitions, not prior evaluated combinations.

check this [documentation](https://www.google.com/url?q=https%3A%2F%2Fkeras.io%2Fkeras_tuner%2Fapi%2Ftuners%2Fbayesian%2F) for details.

```
keras_tuner.BayesianOptimization(
    hypermodel=None,
    objective=None,
    max_trials=10,
    num_initial_points=None,
    alpha=0.0001,
    beta=2.6,
    seed=None,
    hyperparameters=None,
    tune_new_entries=True,
    allow_new_entries=True,
    max_retries_per_trial=0,
    max_consecutive_failed_trials=3,
    **kwargs
)
```

In the above example -
num_initial_points: initialises number of randomly generated samples as initial training data for Bayesian optimization.
hyperparameters: It let predefine a search space , not prior results.

Let me know if this helps!
